docs/index.rst

```diff
@@ -95,3 +95,4 @@ Contents:
    :maxdepth: 3
 
    translating/index
+   [ Index Page ] <https://zulip.readthedocs.io/en/latest/index.html>
```

Comment:
I think we want a relative link, not an absolute one, so it does the right thing in development.

Reply:
yeah I tried but I'm not sure if there's a workaround because index.html doesn't exist until the docs get built so sphinx will complain that `toctree contains reference to nonexisting document 'index.html'` and will simply ignore the link.

Reply:
Hmm, OK.  Let's add a comment explaining that, at least?

Reply:
Oops yes, should've thought that. Done.

Reply:
Can the link just be to `https://zulip.readthedocs.io/en/latest/`?
Also, I might call this "Index" or "Home", especially now that we have a little blurb at the top.

Reply:
Yeah sure, shorter's better

Yeah "Home" would work too, perhaps even "Home/Index"?